react-native-libraries.json

```diff
@@ -9647,7 +9647,10 @@
     "githubUrl": "https://github.com/4cc3ssX/react-native-totp-utils",
     "examples": ["https://github.com/4cc3ssX/react-native-totp-utils/tree/main/example"],
     "ios": true,
-    "android": true
+    "android": true,
+    "newArchitecture": true,
```

Comment:
```suggestion
    "newArchitecture": true,
```

Is this correct?

Reply:
Yes it's correct. Compatible with both new and old architectures while alternative one only supports new architecture.

Reply:
Thanks for confirming! Let's rebase the PR so we can ship it.

Reply:
I've to pull to resolve conflicts and discarded my changes. Creating new PR for now.